openfisca_aotearoa/variables/residence/residence.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Person
+
+
+class normally_lives_in_nz(Variable):
```

Comment:
Needs confirmation I'm linking to the correct reference...
http://legislation.govt.nz/act/public/1964/0136/latest/whole.html#DLM363796
suggests this would be more accurate as: "social_security__ordinarily_resident_in_new_zealand"

Reply:
It came from here:
https://www.workandincome.govt.nz/products/a-z-benefits/accommodation-supplement.html

```
Who can get it

You may get an Accommodation Supplement if you:

    have accommodation costs
    are aged 16 years or more
    are a New Zealand citizen or permanent resident
    normally live in New Zealand and intend to stay here
    are not paying rent for a social housing property. (Social housing properties are provided by Housing New Zealand and approved community housing providers.)
```

Reply:
I can see what's in Raap is just "is_nz_resident" which isn't quite the same thing

We can have is_citizen and is_permanent_resident variables instead and be more accurate.

Reply:
@verbman is the grammar of starting a variable with `is_` going to work for us? What's your feelings on this one?

Reply:
"The information provided in this page is a guide only."
I'm re-doing to be the legislation  -not the MSD guide.

There's also clauses about refugees I'll add.